### ISSUE TITLE
Bump into php-7.1+ versions

### ISSUE THREAD

**Issue by peter279k** [ricbra/php-discogs-api#55]
As title, the `php-7.1` versions have been released for the while.

And I think it's time to consider bumping into the `php-7.1+` versions now.

To do this issue, I think we should consider following points:

- Upgrade the necessary dependencies to the current released versions. Such as `Guzzle`.
- Checking the syntax and modify them to be compatible with the `php-7.1+` versions.

**Comment by ricbra** [comment]
Thank you for opening this issue. Is this something you want to work on?

**Comment by peter279k** [comment]
Hi @ricbra, thank you for your reply/concern.

If this issue is accepted, I can work on this :).

**Comment by ricbra** [comment]
Yes sure, I'm not working actively anymore on this so if someone wants to step in 👍

**Comment by w3spi5** [comment]
Is there any progress for this issue?
